src/uu/date/fuzz/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+# spell-checker:ignore libfuzzer
+
```

Comment:
I'm wondering about what the file structure here should be. We could also make a root `fuzz` folder, though I suppose it doesn't matter much with just one util.

Reply:
as you wish :)

Reply:
@tertsdiepraam 
maybe we can do it when we have a second fuzzer?

Reply:
I'm only now seeing this but yes that sounds good